onward/app/controllers/PopularInTagDCR.scala

```diff
@@ -0,0 +1,41 @@
+package controllers
+
+import common._
+import containers.Containers
+import contentapi.ContentApiClient
+import feed.MostReadAgent
+import model._
+import play.api.libs.ws.WSClient
+import play.api.mvc._
+import renderers.DotcomRenderingService
+import services._
+
+import scala.concurrent.Future
+
+class PopularInTagDCR(
+    val contentApiClient: ContentApiClient,
+    val mostReadAgent: MostReadAgent,
+    val controllerComponents: ControllerComponents,
+    val wsClient: WSClient,
+)(implicit context: ApplicationContext)
+    extends BaseController
+    with Related
+    with Containers
+    with GuLogging
+    with ImplicitControllerExecutionContext
+    with GetPopularInTagDCR
+    with implicits.Requests {
+
+  val remoteRenderer: DotcomRenderingService = DotcomRenderingService()
+
+  def render(tag: String): Action[AnyContent] = {
+    Action.async { implicit request =>
+      val edition = Edition(request)
+      val excludeTags = request.queryString.getOrElse("exclude-tag", Nil)
```

Comment:
this follows the existing pattern for the `popular-in-tag` endpoint. Using this pattern will require us to add the exclude params to the query string in the `OnwardPicker` recreating [the existing logic used in DCR](https://github.com/guardian/dotcom-rendering/blob/cf0942c5e212e709e3adc2f96b725d36574e38aa/dotcom-rendering/src/web/components/OnwardsUpper.importable.tsx#L236-L255).

Reply:
Hopefully we can move this logic to the server soon.